doc/kan.tex

```diff
@@ -114,11 +198,64 @@ \section{Type structure of the universe}
     \IsTp{\TpEl{\Code{A}}}
   }
   \and
+  \inferrule[pi code]{
+    \Code{A} : \TpUniv\\
+    \Code{B} : \TpEl{\Code{A}}\to {\TpUniv}
```

Comment:
Nitpick: in the rest of the document, we're writing lambdas for arguments of function type, but the second argument of the pi and sigma codes are being written as open terms. I think it would be most consistent to change this premise to`x:\TpEl{\Code{A}} \vdash B(x) : \TpUniv`. Thoughts?

Reply:
1. The premise should not be changed.
2. The notation is well-typed as-is, since `B : el(A) -> univ` and therefore `B(x) : univ`; it might be more clear to write `Pi(A,B)` or whatever, but it seems like this notation makes the rest of the code easier to display, so I don't favor making that change.

Reply:
I am neutral. I see the point but am comfortable with implicit beta or eta rules here.